packages/cli/cli/src/commands/validate/logViolations.ts

```diff
@@ -52,13 +54,15 @@ function groupViolationsByNodePath(violations: ValidationViolation[]): Map<NodeP
 
 function logViolationsGroup({
     logWarnings,
-    relativeFilepath,
+    logBreadcrumbs,
+    configRelativeFilepath,
```

Comment:
I renamed `relativeFilepath` to `configRelativeFilepath` because this value was always `docs.yaml` and not the actual filepath that these violations came from.

Let me know if there's a better name or I should change it back.